src/components/Header/Header.tsx

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-import TopCodeMenu from '../Menu/TopCodeMenu';
-import { VersionMenuProps } from '../Menu/VersionMenu';
 import { TopMainNav } from './TopMainNav';
 
-const Header = ({ languages, versionData }: { languages: string[]; versionData: VersionMenuProps }) => {
+export const Navigation = () => {
```

Comment:
Might be worth leaving this called `Header` otherwise we `import { Navigation } from 'components/Header'`.

Seems the component is still 90% Nav anyway 👍